libnavigation-core/build.gradle

```diff
@@ -39,6 +47,8 @@ dependencies {
     implementation project(':libtrip-notification')
     implementation dependenciesList.mapboxSdkTurf
     implementation dependenciesList.mapboxAndroidAccounts
+    implementation dependenciesList.mapboxEvents
+    implementation project(':liblogger')
```

Comment:
Let's not depend on the `liblogger` module. We have to remove it anyway and wait for https://github.com/mapbox/mapbox-base-android/pull/21 to be released. Please use `android.util.Log` until then.

Reply:
👍